lockbox-ios/Presenter/ItemListPresenter.swift

```diff
@@ -371,3 +332,75 @@ extension ItemListPresenter {
         }
     }
 }
+
+extension ItemListPresenter {
```

Comment:
just curious: why have an extension for `ItemListPresenter` instead of adding these functions to the class directly?

Reply:
I've been separating the `fileprivate` smaller functions into their own extension for organizational purposes :p